spec/interactors/save_listing_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe SaveListing do
+  let(:service_area)    { create :service_area }
+  subject(:interaction) { SaveListing.run params }
+
+  describe 'validation' do
+    let(:params) {{
+      type: '',  # only Listing field with validation
+      service_area: service_area.id,
+      person: {
+        preferred_contact_method: 'email',
+        email: 'we@together.coop',
+        location: {
+          city: 'Lansing',
+          state: 'MI',
+        },
+      },
+    }}
+
+    it { is_expected.to be_invalid }
+
+    it 'does not create any records' do
+      expect { interaction }
```

Comment:
I feel like I only vaguely understand why testing the success case would be unnecessary here. Like, I think it's covered by everything else, but I'm having a little trouble following that line of thought today

Reply:
I think you're right that its worth adding a happy path example here ✅.
Will add on the next pass through.